File: version_3/code-samples/ClientApi/Commands/Transformers/Get.cs
namespace Raven.Documentation.CodeSamples.ClientApi.Commands.Transformers
{
	using Raven.Abstractions.Indexing;
	using Raven.Client.Document;

	public class Get
	{
		private interface IFoo
		{
			#region get_1_0
			TransformerDefinition GetTransformer(string name);
			#endregion

			#region get_2_0
			TransformerDefinition[] GetTransformers(int start, int pageSize);
			#endregion
		}

		public Get()
		{
			using (var store = new DocumentStore())
			{
				#region get_1_1
				var transformer = store
					.DatabaseCommands
					.GetTransformer("Order/Statistics"); // returns null if does not exist
				#endregion

				#region get_2_1
				var transformers = store
					.DatabaseCommands
					.GetTransformers(0, 128);
				#endregion
			}
		}
	}
}

File: Documentation/3.0/Samples/java/src/test/java/net/ravendb/clientapi/commands/transformers/Get.java
package net.ravendb.clientapi.commands.transformers;

import java.util.List;
import net.ravendb.abstractions.indexing.TransformerDefinition;
import net.ravendb.client.IDocumentStore;
import net.ravendb.client.document.DocumentStore;

public class Get {

  @SuppressWarnings("unused")
  private interface IFoo {
    //region get_1_0
    public TransformerDefinition getTransformer(String name);
    //endregion

    //region get_2_0
    public List<TransformerDefinition> getTransformers(int start, int pageSize);
    //endregion
  }

  @SuppressWarnings("unused")
  public Get() throws Exception {
    try (IDocumentStore store = new DocumentStore()) {
      //region get_1_1
      TransformerDefinition transformer = store
        .getDatabaseCommands()
        .getTransformer("Order/Statistics"); // returns null if does not exist
      //endregion

      //region get_2_1
      List<TransformerDefinition> transformers = store.getDatabaseCommands().getTransformers(0, 128);
      //endregion
    }
  }
}
